pom.xml

```diff
@@ -83,7 +83,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <dropwizard.version>0.7.1</dropwizard.version>
-        <protobuf.version>2.6.1</protobuf.version>
+        <protobuf.version>2.4.1</protobuf.version>
```

Comment:
I'd like to keep the minimum version 2.6.1 at this point since that's what has been released.